src/config/serverConfigDefaults.ts

```diff
@@ -175,6 +175,8 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     saml_logout_local: false,
     patient_view_use_legacy_timeline: false,
     enable_request_body_gzip_compression: false,
+
+    referenceGenomeVersion: 'hg19',
```

Comment:
seems kinda weird to have this property, since one portal can have multiple reference genomes